listenstore/bin/listenstore-kafka-to-cassandra-writer.py

```diff
@@ -13,6 +15,12 @@ def __init__(self):
     def run(self):
         self.kafkaConsumer.start_listens(self.listenStore)
 
+    @property
+    def listenStore(self):
```

Comment:
There's a catch :'(
All the methods which are marked as `@property` follow CamelCase notation in the existing LB structure. (ex. listenStore, kafkaConsumer). This might violate PEP8 standards, but i believe its good to keep it the way it is. Sometimes even i find it hard to track variables and methods, considering so diverse structure of ListenBrainz.

Though, changing it will require considerable change in a lot of method names.